guest/rust/examples/basics/physics/Cargo.toml

```diff
@@ -9,10 +9,10 @@ version = {workspace = true}
 [dependencies]
 ambient_api = {workspace = true}
 
-[[bin]]
-name = "physics_client"
-path = "src/client.rs"
-required-features = ["client"]
+# [[bin]]
+# name = "physics_client"
+# path = "src/client.rs"
+# required-features = ["client"]
```

Comment:
Shouldn't be commented out; I'd suggest just disabling WASM loading entirely on the client for now